tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Session.java

```diff
@@ -293,6 +293,16 @@ public Runner addTarget(Op op) {
       return addTarget(op.op());
     }
 
+    /**
+     * Make {@link #run} execute the graph's initializers.
+     *
+     * @return this session runner
+     */
+    public Runner doInitialization(){
```

Comment:
This should be called `addInitialization`, and I'm not sure it's a great idea to do this in a runner with other things, so it might want to be a package-private method only visible through the `runInit` method, then fix the docs for `runInit` so it doesn't reference a private method.

Reply:
I'll just remove it if we don't want it public, I wasn't using it in `runInit`